demos/README.md

```diff
@@ -0,0 +1,35 @@
+![buildah logo](https://cdn.rawgit.com/projectatomic/buildah/master/logos/buildah-logo_large.png)
+
+# Buildah Demos
+
+The purpose of these demonstrations is twofold:
+
+1. To help automate some of the tutorial material so that Buildah newcomers can walk through some of the concepts.
+2. For Buildah enthusiasts and practitioners to use for demos at educational presentations - college classes, Meetups etc.
+
+It is assumed that you have installed Buildah and Podman on you machine.
+
+    $ sudo dnf -y install podman buildah
```

Comment:
might want to switch to yum instead of dnf here, either will work on Fedora and yum will also work on RHEL and others.